### DISCUSSION TITLE
Examples of open source projects using gauge?

### DISCUSSION THREAD

**Discussion by dbaggott** [getgauge/gauge#1878] [General]
Hi, are there any good examples of open source projects using gauge?

I'm interested in seeing examples of mature gauge-based automation projects to see what I can learn from their approaches.  I'm especially interested in projects that are using Gauge to test APIs (without browsers/UI involved) but, really, I'm interested in anything.

Thanks!

**Comment by zabil** [comment]
The bahmni project uses Gauge at https://github.com/Bahmni/bahmni-gauge

**Comment by sriv** [comment]
Gauge itself uses gauge to test it's feature - https://github.com/getgauge/gauge-tests

Another product GoCD uses gauge too - https://github.com/gocd/functional-tests

**Reply by johnboyes** [reply]
@dbaggott if it is REST APIs that you're interested in using Gauge to test then have a look at this proof of concept I put together recently: https://github.com/agilepathway/gauge-openapi-example
It's just a small proof of concept but the client I'm working with have been using it successfully (internally, not open source) for the last few months and finding lots of benefits.
Feedback or questions on this approach are welcome :-)
